### ISSUE TITLE
[pytest-xdist] EOFError During handling of the above exception, another exception occurred:

### ISSUE THREAD

**Issue by petrisorionel** [SergeyPirogov/webdriver_manager#290]
```
# Debian 11 with Python 3.9 (3.9.9)
FROM python:3.9-slim-bullseye

# Google Chrome, can specify versions by CHROME_VERSION;
# Latest (equivalent to google-chrome-stable)
ARG CHROME_VERSION="google-chrome-stable"
RUN apt-get update && \
    apt-get install -y gnupg wget curl unzip --no-install-recommends && \
    wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - &&  \
    echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list && \
    apt-get update -qqy &&  \
    apt-get -qqy install \
      ${CHROME_VERSION:-google-chrome-stable} &&  \
    rm /etc/apt/sources.list.d/google-chrome.list &&  \
    rm -rf /var/lib/apt/lists/* /var/cache/apt/*
```

```
------------------------------------------------------------------------------
[gw0] linux -- Python 3.9.10 /usr/local/bin/python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/archive.py", line 38, in __extract_zip
    archive.extractall(to_directory)
  File "/usr/local/lib/python3.9/zipfile.py", line 1633, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "/usr/local/lib/python3.9/zipfile.py", line 1688, in _extract_member
    shutil.copyfileobj(source, target)
  File "/usr/local/lib/python3.9/shutil.py", line 205, in copyfileobj
    buf = fsrc_read(length)
  File "/usr/local/lib/python3.9/zipfile.py", line 922, in read
    data = self._read1(n)
  File "/usr/local/lib/python3.9/zipfile.py", line 990, in _read1
    data += self._read2(n - len(data))
  File "/usr/local/lib/python3.9/zipfile.py", line 1025, in _read2
    raise EOFError
EOFError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\user\workspace\project\conftest.py", line 25, in web_driver_session
  File "/project/driver.py", line 48, in start_web_driver
    service=ChromeService(executable_path=ChromeDriverManager().install()))
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/chrome.py", line 32, in install
    driver_path = self._get_driver_path(self.driver)
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/manager.py", line 31, in _get_driver_path
    binary_path = self.driver_cache.save_file_to_cache(file, browser_version,
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/driver_cache.py", line 28, in save_file_to_cache
    files = archive.unpack(path)
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/archive.py", line 30, in unpack
    return self.__extract_zip(directory)
  File "/usr/local/lib/python3.9/site-packages/webdriver_manager/archive.py", line 40, in __extract_zip
    if e.args[0] not in [26, 13] and e.args[1] not in ['Text file busy', 'Permission denied']:
IndexError: tuple index out of range

```

This happens when I run tests in parallel with pytest-xdist.
my understanding is that there is an instance of chromedriver which is already running, which is possible as I can run this code in the background. How can I handle this error?

@SergeyPirogov

**Comment by aleksandr-kotlyar** [comment]
@petrisorionel Please add [pytest-xdist] into start of the issue's summary. It's specific case because of parallelization. May be someone with expertise will notice it and help to resolve.

**Comment by GonadiusPrime** [comment]
I see the above style of error whenever a new webdriver is released.  I imagine with 4 spawned processes by xdist, and each of them trying to unzip the same file, there will be conflicts as such.

**Comment by aleksandr-kotlyar** [comment]
Yes it does. Need to resolve this issue.

**Comment by Karamann** [comment]
What wdm version do you use @petrisorionel ?
We face the same issue with xdist + 3.5.2 version of wdm.

**Comment by SergeyPirogov** [comment]
That's because every thread tries to download the driver and put it in the same folder. Need to think about fix

**Comment by SergeyPirogov** [comment]
@petrisorionel Hello, it's fixed in v3.7.0.
